components/about-us/aboutuscard.vue

```diff
@@ -88,14 +89,14 @@
       <div class="text <lg:(w-3/4 mt-5 h-1/3 text-center) w-1/2">
         <h1 class="mb-5px font-medium text-2xl text-gray-50">WHAT WE DO?</h1>
         <p class="text-gray-300">
-          Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nullam tortor
-          eros, consequat sit amet euismod rutrum, ultricies ut dui. Donec et
-          dui non lorem tincidunt scelerisque ut eu lacus. Nullam a nunc in nisi
-          varius suscipit at vitae ante. Nulla magna ex, laoreet a molestie eu,
-          sagittis id augue. Sed molestie hendrerit ex eu blandit. Nam ut
-          imperdiet purus. Fusce pretium sollicitudin dapibus. Etiam gravida
-          fermentum orci semper luctus. Maecenas est arcu, sagittis non purus
-          vitae, blandit egestas nibh.
+          Whether Mobile Development, Desktop Development, Web Development, Game
+          Development, or Cloud Development, we manufacture all units here!
+          Along the lines of development, we enjoy discussions and
+          collaborations too. Regular DevTalks for discussions and doubts about
+          new tech and associating over Open Source provides us with great
+          opportunities of expanding our colossal network. DevOps with GitLab,
```

Comment:
Let's change devops with gitlab to multiverse of languages

Reply:
Okay, will update it!!